test/integration/select_test.lua

```diff
@@ -805,3 +823,34 @@ add('test_select_with_collations', function(g)
     local objects = crud.unflatten_rows(result.rows, result.metadata)
     t.assert_equals(objects, get_by_ids(customers, {2, 4}))
 end)
+
+add('test_select_from_space_with_uuid_pk', function(g)
+    local goods = insert_goods(g, {
+        {
+            uuid = uuid:new(), name = "IPhone 12"
+        },
+        {
+            uuid = uuid:new(), name = "Samsung S11"
+        },
+        {
+            uuid = uuid:new(), name = "Nubia Z11"
+        },
+        {
+            uuid = uuid:new(), name = "Google Pixel 3"
+        },
+    })
+
+    table.sort(goods, function(obj1, obj2) return obj1.uuid:str() < obj2.uuid:str() end)
+
+    local result, err = g.cluster.main_server.net_box:call('crud.select', {'goods', nil})
+    t.assert_equals(err, nil)
+    local objects = crud.unflatten_rows(result.rows, result.metadata)
+    t.assert_equals(#objects, 4)
+
+    local conditions = {{'>', 'uuid', goods[3].uuid}}
```

Comment:
There is no test where field with uuid type is additional check

Reply:
Added additoinal checks.